screens/LoginScreen.js

```diff
@@ -408,218 +345,201 @@ class LoginScreen extends React.Component {
     const domainErrorMessage = domainValidation ? <Text style={styles.validationErrorMessage}>{i18n.t('login.domain.error')}</Text> : null;
     const userErrorMessage = userValidation ? <Text style={styles.validationErrorMessage}>{i18n.t('login.username.error')}</Text> : null;
     const passwordErrorMessage = passwordValidation ? <Text style={styles.validationErrorMessage}>{i18n.t('login.password.error')}</Text> : null;
-
+*/
     const languagePickerItems = locales.map(locale => (
       <Picker.Item label={locale.name} value={locale.code} key={locale.code} />
     ));
     return (
-      <KeyboardAvoidingView behavior="padding">
-        <ScrollView contentContainerStyle={styles.container}>
-          <View style={styles.header}>
-            <Image
-              source={require('../assets/images/dt-logo2.png')}
-              style={styles.welcomeImage}
-            />
-          </View>
-
-          <View style={styles.formContainer}>
-            <TextField
-              containerStyle={domainStyle}
-              iconName="ios-globe"
-              label={i18n.t('login.domain.label')}
-              onChangeText={text => this.setState({ domain: text })}
-              autoCapitalize="none"
-              autoCorrect={false}
-              value={this.state.domain}
-              returnKeyType="next"
-              textContentType="URL"
-              disabled={this.state.loading}
-              placeholder={i18n.t('login.domain.placeholder')}
-            />
-            {domainErrorMessage}
-
-            <TextField
-              containerStyle={userStyle}
-              iconName={Platform.OS === 'ios' ? 'ios-person' : 'md-person'}
-              label={i18n.t('login.username.label')}
-              onChangeText={text => this.setState({ username: text })}
-              autoCapitalize="none"
-              autoCorrect={false}
-              value={this.state.username}
-              returnKeyType="next"
-              textContentType="emailAddress"
-              disabled={this.state.loading}
-            />
-            {userErrorMessage}
-
-            <TextField
-              containerStyle={passwordStyle}
-              iconName={Platform.OS === 'ios' ? 'ios-key' : 'md-key'}
-              label={i18n.t('login.password.label')}
-              onChangeText={text => this.setState({ password: text })}
-              autoCapitalize="none"
-              autoCorrect={false}
-              secureTextEntry
-              value={this.state.password}
-              returnKeyType="go"
-              selectTextOnFocus
-              onSubmitEditing={this.signInAsync}
-              blurOnSubmit
-              textContentType="password"
-              disabled={this.state.loading}
-            />
-            {passwordErrorMessage}
-
-            {!this.state.loading && (
+      <View style={styles.container}>
```

Comment:
I think this was accidentally un-done as part of the merge. It needs to be the KeyboardAvoidingView and ScrollView to ensure the fields stay within the viewable area when the keyboard is shown (especially on smaller devices).

Reply:
Sorry for that, i will to re-implement this.